k8s/mongodb/container/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM mongo:3.4.10
+FROM mongo:3.4.13
 LABEL maintainer "dev@bigchaindb.com"
 WORKDIR /
 RUN apt-get update \
     && apt-get -y upgrade \
     && apt-get autoremove \
     && apt-get clean
-COPY mongod.conf.tmt.template /etc/mongod.conf
-COPY mongod_tmt_entrypoint.bash /
+COPY mongod.conf.template /etc/mongod.conf
```

Comment:
Did you do a grep for all occurrences of `tmt`?

Reply:
yes.

Reply:
:+1: Yay grep!